applications/DEMApplication/test_examples/DEM_run_all_benchmarks_analysis.py

```diff
@@ -84,10 +85,8 @@ def Run():
                 if platform.system()=="Windows":
                     subprocess.check_call(["python", path + "/DEM_benchmarks_analysis.py", str(benchmark), ">", "BenchTemp.info"], stdout=f, stderr=f)
```

Comment:
I don't think you still need this version when using what I suggested

Reply:
fixed in two places